llama/generation.py

```diff
@@ -79,6 +79,17 @@ def build(
         """
         model_parallel_size = 1
 
+        local_rank = int(os.environ.get("LOCAL_RANK", 0))
```

Comment:
you can just delete it, I nuked all distributed stuff as we're doing shard-by-expert from a single process

Reply:
cleaned this up